Status.md improvements, other fixes (#378)
erdilS/Port-Windows-11-Xiaomi-Pad-5: guide/English/status.md

| Feature                | Notes                                           | Status         |
|------------------------|-------------------------------------------------|----------------|
| 🔊 Audio               |                                                 | ✅            |

> | 🔋 Battery             | Sometimes it may give incorrect values                                                | ✅            |

Reviewer: Wasn't the battery % fixed? Or is it still showing the wrong values on some NABUs?

Author: > Wasn't the battery % fixed? Or is it still showing the wrong values on some NABUs?

It can give the wrong value sometimes, now it just doesn't beat as much

Reviewer: ```suggestion
| 🔋 Battery             | May occasionally report incorrect values                                                | ✅            |

| Feature                         | Notes                                                            | Status         |
|---------------------------------|------------------------------------------------------------------|----------------|
| 🪵 USB-Fn (Charging & MTP)     | **[Default]**                                                     | ✅             |

> | 🪵 USB-Host (OTG)              | Sometimes it may not work until you reboot or reenable USB-Host   | ✅             |

Reviewer: ```suggestion
| 🪵 USB-Host (OTG)              | May require a system reboot or toggling USB Host mode to function | ⚠️            |

Author: че блять? Я извиняюсь ты сам понял что написал?

> | 🪵 USB-Host (OTG)              | May require a system reboot or toggling USB Host mode to function | ⚠️            |

Reviewer 2: this is a lot better.

| Feature                         | Notes                                                            | Status         |
|---------------------------------|------------------------------------------------------------------|----------------|
| 🪵 USB-Fn (Charging & MTP)     | **[Default]**                                                     | ✅            |

> | 🪵 USB-Host (OTG)              | It may not work, you need to reboot or [this instruction](https://github.com/erdilS/Port-Windows-11-Xiaomi-Pad-5/blob/main/guide/English/Additional-materials-en.md#toggling-usb-host-mode) | ✅|

Reviewer: ```suggestion
| 🪵 USB-Host (OTG)              | May require a system reboot or toggling USB Host mode to function | ⚠️            |
```

Author: The USB host works fine, it just requires a reboot sometimes

Reviewer: > The USB host works fine

It doesn't

Author: The function is in development, which function is in development? USB host works perfectly with and without additional power supply.

Author: > > The USB host works fine
> 
> It doesn't

Why?

Reviewer 2: you need to revert that. (commit 14edb20 on your fork), it looks better.

Author: > you need to revert that. (commit [14edb20](https://github.com/erdilS/Port-Windows-11-Xiaomi-Pad-5/commit/14edb20bf743f5bb5cb8fbdc3f1bdf000e760874) on your fork), it looks better.

I'm not implying anything, but it a lie. Is this feature in development? OTG without power supply works fine, it just requires a reboot sometimes

Author: > you need to revert that. (commit [14edb20](https://github.com/erdilS/Port-Windows-11-Xiaomi-Pad-5/commit/14edb20bf743f5bb5cb8fbdc3f1bdf000e760874) on your fork), it looks better.

awhwhh sorry. im not sure I can do this because, as I understand it, its written quite poorly - I've paraphrased it

| Feature                | Notes                                       | Status         |
|------------------------|---------------------------------------------|----------------|
| 🛡️ TPM                 |                                             | ❌            |
| 🛡️ Security processor  | Unavailable                                 | ❌              |
| 🔒 BitLocker           | Available but not recommended               | ✅            |
| 🛡️ Secure Boot         |                                             | ✅            |

> | 🛡️ Windows Hello Biometrics      |   | ❌            |

Reviewer 2: you can use the old one for giving users information about why not works so deleting this is not seems good

Reviewer: Like this maybe?
```suggestion
| 🛡️ Windows Hello Biometrics      | Requires a fingerprint scanner  | ❌            |

Author: windows hello also work with camera. But camera didnt work and doesnt support

| 🧭 Gyroscope           |                                             | ✅            |
| 🧭 Light sensor        |                                             | ✅            |
| 🧭 Magnetometer        |                                             | ✅            |

> | 🧭 Proximity           |                                             | ✅            |

Reviewer 2: why removing proximity?

Author: > why removing proximity?
> 

@dizaumuna it doesn't exist in the tablet

Reviewer 2: it does. revert that too.

Author: > it does. revert that too.

@dizaumuna it doesnt exist

Author: <img width="2560" height="1600" alt="image" src="https://github.com/user-attachments/assets/20de1283-45d8-4f16-8adf-9d1a8c72c4bc" />
приближение = proximity

Reviewer: <img width="845" height="167" alt="image" src="https://github.com/user-attachments/assets/50869fe8-f3bc-4d16-a4cc-eb12e3907555" />

Author: > <img alt="image" width="845" height="167" src="https://private-user-images.githubusercontent.com/118528504/571958335-50869fe8-f3bc-4d16-a4cc-eb12e3907555.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NzQ5Njk0NzYsIm5iZiI6MTc3NDk2OTE3NiwicGF0aCI6Ii8xMTg1Mjg1MDQvNTcxOTU4MzM1LTUwODY5ZmU4LWYzYmMtNGQxNi1hNGNjLWViMTJlMzkwNzU1NS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjYwMzMxJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI2MDMzMVQxNDU5MzZaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1lNjFhY2QyZjM4ZTdkNjM4NWRjZWY2YmM1OTgyYjJmYmE4N2Q4OTkwMDUxNmEwOGZhN2MyYmZiYzRiMGUyMjVhJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.k9tsW-pRdZiJ-h_a4yalq2VpNPDJ6kyLFdK056rXf4Q">

а прога говорит нет такого

Reviewer: > а прога говорит нет такого

I rely on what _documentation_ says

Author: <img width="696" height="445" alt="image" src="https://github.com/user-attachments/assets/e322031a-fe10-41d3-b2df-53a89bec39ae" />
<img width="684" height="518" alt="image" src="https://github.com/user-attachments/assets/e713217a-a576-4398-8456-bfc3f11fcbaf" />

Author: > > а прога говорит нет такого
> 
> I rely on what _documentation_ says

а можешь мне показать _документацию_ где это сказано?
